### ISSUE TITLE
Training on multiple gpus

### ISSUE THREAD

**Issue by chenzhik** [paul007pl/MVP_Benchmark#34]
Dear Authors,

I noticed that you train the model for completion "using the Adam optimizer with initial learning rate 1e−4
(decayed by 0.7 every 40 epochs) and batch size 32 by NVIDIA TITAN Xp GPU" mentioned in your paper. Did you mean that you used a single GPU for training or more GPUs? 

Thanks for your work and look forward to your favourable reply.

**Comment by kobeees** [comment]
Hello, I ran the author's official code to complete the task at 8192 resolution. Found that it takes only 20 minutes to train an epoch, but verification takes 2 hours on an NVIDIA 3090 GPU. Have you ever encountered this situation? Thank you!
